blogs.json

```diff
@@ -3837,8 +3837,8 @@
           {
             "title": "Rudrank's Blog",
             "author": "Rudrank Riyam",
-            "site_url": "https://www.rudrank.com/",
-            "feed_url": "https://www.rudrank.com/rss/",
+            "site_url": "https://error.ghost.org/",
+            "feed_url": "https://error.ghost.org/",
```

Comment:
Hey @rudrankriyam, is this correct? Has your blog shut down?

Reply:
I'll merge this for now to keep the site free of dead links, but if you do get the blog back up and running please do re-add it! 👍